components/facebook_groups/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started
+You need to install the Pipedream app in your Facebook Group in order to make any API calls, or use any prebuilt triggers or actions. To install Pipedream, navigate to the group for which you are an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the Edit button under **Apps**.
+![Apps](https://res.cloudinary.com/dpenc2lit/image/upload/v1687367319/mycology_fb_bpfmh3.png)
+
+Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app has been successfully added to your group, you can start building automated workflows on Pipedream using Facebook Groups.
```

Comment:
```suggestion
Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app is successfully added to your group, you can start building automated workflows on Pipedream using Facebook Groups.
```

Reply:
Changed 'has been' to 'is' - Google's Developer docs style guide (present tense)